src/slider/Slider.ts

```diff
@@ -130,7 +130,7 @@ export default class Slider extends SliderBase<SliderProperties> {
 		const percentValue = (value - min) / (max - min) * 100;
 
 		// custom output node
-		const outputNode = output ? output(value) : value + '';
+		const outputNode = output ? output(value) : `${value}`;
```

Comment:
both of those are ugly... `String(value)` would be so much better... 🙉

Reply:
Template literals for the win 🙈

Reply:
Personally, I prefer `value + []` for all string casting 🙊

Reply:
```
`${value}` + String('')
```

Reply:
```js
typeof value === 'string' ? value : value.toString()
```

Reply:
```ts
[ '' ] + [ value ]
```

🤡

Reply:
```
Math.random() > 0.5 ? `${value + ''}` : String(value);
```

Reply:
Template literals are fastest now (we had a JSPerf page going around another PR at some point.)

Reply:
Ok, fine, though we shouldn't always overvalue performance for code clarity though.

Reply:
Also, there is no speed advantage as long as we continue to emit to ES5, as the down emit would be:

```js
"" + value
```

But even that is more performant than `String(value)`

Reply:
Main reason I like like template literals is that is for consistency as we use them in various scenarios across all our packages. One of which is to effectively stringify a variable :)

Reply:
Using literals to stringify variables, as long as we are consistent is sufficient and clear.  It is a darn sight better than `value + ''`.  Refs: dojo/meta#198